src/App.js

```diff
@@ -12,6 +12,9 @@ function App() {
         <Route path="/Products" element={<Products />} />
         <Route path="/WishList" element={<WishList />} />
         <Route path="/Cart" element={<Cart />} />
+        <Route path="/SignUp" element={<SignUp />} />
+        <Route path="/Login" element={<Login/>} />
+        <Route path="/Logout" element={<Logout/>} />
```

Comment:
Instead of adding routes in the App.js file, you could move it to a separate file say RouteList.js and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner and readable

Reply:
thanks for suggestion, will try to implement it